.env.example

```
@@ -50,3 +50,9 @@ R2_ACCESS_KEY_ID='your_r2_access_key_id'
 R2_SECRET_ACCESS_KEY='your_r2_secret_access_key'
 R2_BUCKET_NAME='ishortn-images'
 R2_PUBLIC_URL='https://pub-xxx.r2.dev'
+
+# Visitor-IP pseudonymization
+# HMAC key used to hash visitor IPs before storing them in UniqueLinkVisit.
+# Any high-entropy random string (>=16 chars). Rotate to reset unique-visit counters.
+# Generate with: openssl rand -hex 32
+IP_HASH_SECRET='change_me_to_a_long_random_string'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)